engine/src/main/scala/cromwell/engine/workflow/tokens/JobTokenDispenserActor.scala

```diff
@@ -197,4 +222,11 @@ object JobTokenDispenserActor {
   @JsonCodec(encodeOnly = true)
   final case class TokenTypeState(tokenType: JobTokenType, queue: TokenQueueState)
 
+  final case class TokenLeaseRecord(tokenLease: Lease[JobToken], time: OffsetDateTime)
+
+  private val tokenDispensorMetricsBasePath: NonEmptyList[String] = MetadataServiceActor.MetadataInstrumentationPrefix :+ "token_distributor"
+  private val requestsEnqueuedMetricPath: NonEmptyList[String] = tokenDispensorMetricsBasePath :+ "requests_enqueued"
+  private val tokensLeasedMetricPath: NonEmptyList[String] = tokenDispensorMetricsBasePath :+ "tokens_distributed"
```

Comment:
TOL ok leased + dispensed, 2 metaphors max

Reply:
👍 switching to "dispensed"

Reply:
Also noticed that the root was basing itself on the `metadata` path prefix which seems.... questionable. So I'm removing that too.